ISISScientificComputing/autoreduce
# 419 system sql queries

### Summary of work
Creation of system performance queries script to perform queries to gather run statistics

### Where
scripts/system_performance/system_performance_queries.py
scripts/system_performance/tests/tests_system_performance_queries.py

### How to test your work
• Check code quality is acceptable
• Ensure all unit tests pass
* Ensure there is no drop in coverage

Fixes #419
